Makefile

```diff
@@ -83,7 +83,7 @@ lint:
 	typos
 	markdownlint '**/*.md'
 	yamllint --no-warnings .
-	shellcheck --format=gcc ${BASH_SCRIPTS}
+	shellcheck --nocomments --format=gcc ${BASH_SCRIPTS}
```

Comment:
Could this be put into another PR?

Reply:
I cannot remember now why this was added.